test/fixtures/basic/main.tf

```diff
@@ -23,4 +23,5 @@ module "datalab" {
   project_id         = var.project_id
   datalab_user_email = var.datalab_user_email
   create_fw_rule     = false
+  service_account    = "project-service-account@${var.project_id}.iam.gserviceaccount.com"
```

Comment:
This address should be propagated from the `setup` configuration via its outputs and a variable input here.